### DISCUSSION TITLE
paperspace blocked my account and sent me this email, beware, may happen to you

### DISCUSSION THREAD

**Discussion by Headupdef** [TheLastBen/fast-stable-diffusion#2128] [General]
We've detected some unusual activities in your notebook and want to ensure everything is okay. We apologize for any inconvenience this may have caused, and we're here to help you resolve this issue.

To move forward, we need to verify your account manually. This standard security measure protects your account and ensures it is used for legitimate purposes, such as running experiments, testing software, or developing applications.

To complete the verification process, please provide us with some information about your intentions using our product and additional details such as your contact information, company or personal website, and a link to your GitHub or social media account.

Just so you know, certain activities, such as using our platform for mining cryptocurrency, tunneling, SSH, NGROK, or RDP, are strictly prohibited and can result in the suspension or termination of your account.

If you have any questions, please don't hesitate to contact us. We're here to help and look forward to hearing from you.

Best regards,

**Comment by TheLastBen** [comment]
I'll contact Paperspace

**Comment by Headupdef** [comment]
I spoke to a man, and explained what I was using it for and they reinstated
my account, however I notice all the free gpu are now blacked out for me
even though I subscribed to pro. It wants me to use gpu for money per hr

**Comment by Headupdef** [comment]
ahh nvm i see it blacks out when its full, but yeah he said ngrok tunnels
arent allowed

**Reply by TheLastBen** [reply]
I'll find a solution for the tunnel

**Comment by TintaViva** [comment]
Hi, i'm new in this, what Template is one i need to use for using automatic 1111?
![image](https://user-images.githubusercontent.com/120289853/236919607-c984ebd0-d5da-4bbf-8d09-89c06a62bc85.png)

**Reply by TheLastBen** [reply]
https://console.paperspace.com/github/TheLastBen/PPS
